src/ui/image.cpp

```diff
@@ -211,9 +214,46 @@ ImageBuffer *get_image(unsigned bid) {
   return result;
 }
 
+void scaleImage(ImageBuffer* image, var_num_t scaling) {
+  if(scaling == 1.0 || scaling <= 0.0) {
```

Comment:
could you please change all the occurrences `"if("` to `"if ("` (ie add the space)

Reply:
done.